litellm/utils.py

```diff
@@ -6241,6 +6241,8 @@ def get_supported_openai_params(
     - List if custom_llm_provider is mapped
     - None if unmapped
     """
+    if not custom_llm_provider:
+        custom_llm_provider = litellm.get_llm_provider(model=model)[1]
```

Comment:
`get_llm_provider` raises an exception if the model's llm provider cannot be inferred 

'get_supported_openai_params' returns None for the same case. 

We should make sure it's try-excepted correctly to handle that @jamesbraza

Reply:
Gotchu sounds good, implemented this with a test case too 👍